vault/resource_mount_test.go

```diff
@@ -107,6 +107,26 @@ func TestResourceMount_SealWrap(t *testing.T) {
 	})
 }
 
+// Test Audit HMAC Request Keys attributes
+
+func TestResourceMount_AuditNonHMACRequestKeys(t *testing.T) {
+	path := "example-" + acctest.RandString(10)
+	resource.Test(t, resource.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testResourceMount_InitialConfigAuditNonHMACRequestKeys(path),
+				Check:  testResourceMount_InitialCheckAuditNonHMACRequestKeys(path),
```

Comment:
Suggest checking the state attributes as well.
```suggestion
				Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr("vault_mount.test", "path", path),
					resource.TestCheckResourceAttr("vault_mount.test", "audit_non_hmac_request_keys.#", "2"),
					resource.TestCheckResourceAttr("vault_mount.test", "audit_non_hmac_request_keys.0", "test1request"),
					resource.TestCheckResourceAttr("vault_mount.test", "audit_non_hmac_request_keys.1", "test2request"),
					resource.TestCheckResourceAttr("vault_mount.test", "audit_non_hmac_response_keys.#", "2"),
					resource.TestCheckResourceAttr("vault_mount.test", "audit_non_hmac_response_keys.0", "test1response"),
					resource.TestCheckResourceAttr("vault_mount.test", "audit_non_hmac_response_keys.1", "test2response"),
					testResourceMount_CheckAuditNonHMACRequestKeys(
						path,
						[]string{"test1request", "test2request"},
						[]string{"test1response", "test2response"}),
				),
```